### ISSUE TITLE
Adding "new section heading"

### ISSUE THREAD

**Issue by quochung2310** [academic-moodle-cooperation/moodle-mod_offlinequiz#167]
When I create a normal quiz, I can create more than one section:
![Untitled](https://user-images.githubusercontent.com/85430984/152691851-d861ff2f-a05b-4e04-beb6-2fc08ed4174d.png)

I am using Moodle 3.11
I would like to create a test like this:

Section 1: Topic 1
  1) Question 1
  2) Question 2
  ...
  35)...
Section 2: Topic 2
  36) Question 36
  ....

Also, I would like to print the offline quiz with sections as stated as the above.
Unfortunately, I could not find the option to create new section heading in Offline Quiz.
Did I miss something or this option is discarded?
Could you please let me know how to solve this issue?
Thank you very much for your time!

**Comment by goggo24** [comment]
You are right, at the moment we don't support this function in offline test - because the section headings are not printed. When you want to create headings we use a workaround => add a description question - this is printed out. Did it work?

**Comment by quochung2310** [comment]
Thank you very much for your reply.
Yes, it might work but there is a drawback since I want to shuffle the questions before creating the forms. Also, I would like to have an option to shuffle questions in each sections. 
I think this option is crucial for an offline test, and really hope it will be available in the next updated version.
Appreciate your time and effort!

**Comment by mwreichel** [comment]
Hi @quochung2310,
sorry for our late answer to your further feedback.
We will put it in our backlog.
